Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -20,6 +20,8 @@ public final class StripeCardReaderService: NSObject {
     private let discoveredStripeReadersCache = StripeCardReaderDiscoveryCache()
 
     private var pendingSoftwareUpdate: ReaderSoftwareUpdate?
+
+    private var initialized: Bool = false
```

Comment:
I think we might be able to replace this with a call to `Terminal.hasTokenProvider`?